homeassistant/util/unit_system.py

```diff
@@ -93,6 +93,7 @@ def __init__(
         mass: str,
         pressure: str,
         accumulated_precipitation: str,
+        length_conversions: dict[str | None, str],
```

Comment:
Looking ahead, I think it might make sense to have this based on `device_class`:
`dict[SensorDeviceClass, dict[str | None, str]]`